clang-tools-extra/docs/clang-tidy/checks/readability/implicit-bool-conversion.rst

```diff
@@ -133,3 +133,17 @@ Options
 
    When `true`, the check will allow conditional pointer conversions. Default
    is `false`.
+
+.. option::  UseUpperCaseLiteralSuffix
+
+   When `true`, the replacements will use an uppercase literal suffix in the
+   provided fixes. Default a lowercase literal suffix is used.
+
+    Example
```

Comment:
Personally I don't think this is needed, the above text is already clear enough. But I'm fine to keep it.